msteams-platform/concepts/authentication/resource-specific-consent.md

```diff
@@ -0,0 +1,117 @@
+Add a 'webApplicationInfo' section to the manifest that includes two properties - `id` is the app Id, and `applicationPermissions` are the RSC permissions your app requires.
+
+
+> **Note**: If your app needs non RSC permissions, you can continue to use them as always, but they do not go in the manifest. 
+
+
+```
+{ 
+  "$schema": ".../MicrosoftTeams.schema.json", 
+
+   "webApplicationInfo": { 
+
+      "id": "db5c3ee6-4180-4b81-b080-a3ce61e16b66", 
+
+      "applicationPermissions": [  
+
+          "Channel.Delete.Group", 
+
+          "ChannelMessage.Read.Group", 
+
+          "TeamsApp.Read.Group" 
+      ] 
+  }
+  ... 
+} 
+```  
+Refer list of available Graph permissions for RSC in [Teams permissions](~/permissions-reference.md) section.
+
+## 4. Get an access token 
+
+Before making a REST call to the Graph, you'll need to get a token for application permissions. This is the same as getting an application permission token for non-RSC use. Refer [Get an access token to make a Graph call](~/auth-v2-service) for more details. 
+
+
+  
+```
+string response = await HttpHelpers.POST( 
+
+    $"https://login.microsoftonline.com/{tenant}/oauth2/v2.0/token", 
+
+    $"grant_type=client_credentials&client_id={appId}&client_secret={appSecret}" 
+
+    + "&scope=https%3A%2F%2Fgraph.microsoft.com%2F.default"); 
+
+string token = response.Deserialize<TokenResponse>().access_token; 
+```
+  
+## 5. Make a Graph call 
+
+This works exactly the same as non-RSC Graph calls. You can directly make a REST call or use the SDK. See [Microsoft Graph](https://developer.microsoft.com/en-us/graph) and [Use the Microsoft Graph API to work with Microsoft Teams](~/api/resources/teams-api-overview?view=graph-rest-beta) for more details. 
+
+##  RSC Guidelines
+
+- **Team owner needs a Teams app** : 
+  To provide consent to your app, the team owner needs to install a Teams app. Webpages running outside of Teams may make RSC Graph calls, but only after the corresponding Teams app has been installed.
+
+- **Not every graph API supports RSC** :
+Team owners can only give consent to team data, but many Graph permissions aren't about Team data. It's fine for your app to need Mail.Read, but that's not an RSC scenario because it's not team data, and team owners can't give consent for other people's mailboxes. 1-1 chats and group chats aren't taking place within a team, so RSC does not provide access to that data. Within team data, APIs for Files, SharePoint, OneNote, Planner, and Calendar do not support RSC yet. 
+
+- **Not everyone can install RSC apps** : 
+  Admins can choose to disable RSC (it's enabled by default). Admins may also limit the ability to consent to specific team owners. Team members are not allowed to give consent to Team data, only team owners can provide the consent. 
+Installation for Teams apps is atomic, either the app has consent or the app is not installed. 
+
+- **Don't have multiple Teams apps linked to the same app Id** :
```

Comment:
Recommend clarifying this to "Azure AD app ID", as the "app ID" concept exists for Teams apps themselves (it gets confusing sometimes).

Reply:
Done. Thanks.